app/src/neptune_eye/frame_capture/image_capture.py

```
@@ -0,0 +1,115 @@
+from typing import List, Optional, Tuple
+
+import cv2
+import numpy as np
+
+from .frame_capture_interface import FrameCaptureInterface
+
+
+class ImageCapture(FrameCaptureInterface):
+    """Capture frames by iterating over images in a directory."""
+
+    _SUPPORTED_EXTENSIONS = {".jpg", ".jpeg", ".png", ".bmp", ".tif", ".tiff"}
```

Ruff (0.14.5)

Mutable class attributes should be annotated with `typing.ClassVar`

(RUF012)